dev-test/account.test.js

```diff
@@ -0,0 +1,135 @@
+import path from "path"
+import {describe} from "yargs"
+import {
+  emulator,
+  init,
+  getAccountAddress,
+  getContractAddress,
+  deployContractByName,
+  getScriptCode,
+  executeScript,
+  sendTransaction,
+  mintFlow,
+  getFlowBalance,
+  shallRevert,
+  shallResolve,
+  shallPass,
+  shallThrow,
+  createAccount,
+  HashAlgorithm,
+  SignatureAlgorithm,
+} from "../src"
+import {playgroundImport} from "../src/transformers"
+import * as fcl from "@onflow/fcl"
+import {ec as EC} from "elliptic"
+import {validateKeyPair} from "./util/validate-key-pair"
+
+// We need to set timeout for a higher number, because some transactions might take up some time
+jest.setTimeout(10000)
```

Comment:
Which transactions is this waiting on?

Reply:
None, I had just templated the test file from the existing ones and they all had the increased jest timeout.  This is likely unnecessary and I can remove it.